static/js/jquery.js

```diff
@@ -1,4 +1,2 @@
+/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
Is there a way to add `JQuery.js` as a dependency with some kind of dependencymanagement instead of adding like a `drop binary and go`.

Reply:
i mean npm could be used as dependency management using a requirements.txt

Reply:
Yes sure, but that's question for @zorchenhimer.

Reply:
yeah the reason I just manually updated it is I saw it using 2.x, and it's a single file.